R/functions.R

```diff
@@ -294,6 +294,28 @@ get_projections <- function(job_id) {
   return(list(original = original_projections, avatars = avatar_projections))
 }
 
+#' Get explained variance of the original records.
```

Comment:
If it's about the original records, shouldn't we pass the `dataset_id` instead of the `job_id`?

Reply:
It is indeed about the original records, but we can do multiple avatarizations with one dataset, and the explained variance gets computed in the model, which takes `ncp` and `column_weights`, so we do need the job to get these parameters, and thus access to the model.